src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php

```diff
@@ -150,7 +155,7 @@ protected function getPackageString(PackageCollection $packageCollection): strin
     protected function createResponse(Process $process): ResponseDto
     {
         $command = str_replace('\'', '', $process->getCommandLine());
-        $output = $process->getExitCode() ? $process->getErrorOutput() : '';
+        $output = $process->isTerminated() && $process->isSuccessful() ? $process->getErrorOutput() ?: $process->getOutput() : '';
```

Comment:
Something is wrong here. We should output error if the process is not successful